src/DataProtection/DataProtection/src/Repositories/FileSystemXmlRepository.cs

```diff
@@ -16,7 +17,7 @@ namespace Microsoft.AspNetCore.DataProtection.Repositories;
 /// <summary>
 /// An XML repository backed by a file system.
 /// </summary>
-public class FileSystemXmlRepository : IXmlRepository
```

Comment:
@halter73 Do I need to keep implementing the old interface as well for back compat?

Reply:
What you have is fine.

> It is not breaking when you added the implementation of an interface which derives from the removed interface. For example, you removed `IDisposable`, but implemented `IComponent`, which derives from `IDisposable`.

[breaking-change-rules.md](https://github.com/dotnet/runtime/blob/856a0a6f5e991fd18e125bcfc88741d919688fbf/docs/coding-guidelines/breaking-change-rules.md#L162)